### PR TITLE
luvit 2.6.0

### PR DESCRIPTION
**Pull request by number5**
bumped to latest version

### CODE REVIEW

File: Library/Formula/luvit.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   revision 1
 
   bottle do
     cellar :any
-    sha256 "3c7250314f67d320d8bc36f14e8d055860a2098bf5528b53eb58ff7d93244881" => :yosemite
-    sha256 "1f4616aa1be7802088900bcec2f284e8a85a4b82ee6fc9873a4c20d30ac6ac47" => :mavericks
-    sha256 "ef23e4e0cf252074738d85b1b2b2e02fd58e3a5239e00362d6f1e4dc2c58862c" => :mountain_lion
+    sha256 "" => :yosemite
+    sha256 "" => :mavericks
+    sha256 "" => :mountain_lion
```
Comment (mikemcquaid) [inline Library/Formula/luvit.rb]: Leave these as-is; they will be automatically updated when we `brew pull`.

File: Library/Formula/luvit.rb [hunk 2]
```diff
@@ -1,16 +1,16 @@
 class Luvit < Formula
   desc "Asynchronous I/O for Lua"
   homepage "https://luvit.io"
-  url "https://luvit.io/dist/latest/luvit-0.8.2.tar.gz"
-  sha256 "c2639348d1716c38ac3cd66ea4c4ff1c8a72f4610dbd6e50cf31426d3956c5ff"
+  url "https://github.com/luvit/luvit/archive/2.6.0.tar.gz"
+  sha256 "d6ed70ecf58a52130449a6dadbf7615514feeaf98f17034ff97772cbda6592d1"
   head "https://github.com/luvit/luvit.git"
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/luvit.rb]: Remove the `revision` for version bumps.

**Comment by mikemcquaid** [comment]
Two comments then should be good to go!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

File: Library/Formula/luvit.rb [hunk 3]
```diff
@@ -1,10 +1,10 @@
 class Luvit < Formula
   desc "Asynchronous I/O for Lua"
   homepage "https://luvit.io"
-  url "https://luvit.io/dist/latest/luvit-0.8.2.tar.gz"
-  sha256 "c2639348d1716c38ac3cd66ea4c4ff1c8a72f4610dbd6e50cf31426d3956c5ff"
+  url "https://github.com/luvit/luvit/archive/2.6.0.tar.gz"
+  sha256 "d6ed70ecf58a52130449a6dadbf7615514feeaf98f17034ff97772cbda6592d1"
   head "https://github.com/luvit/luvit.git"
-  revision 1
+
```
Comment (mikemcquaid) [inline Library/Formula/luvit.rb]: For future reference you can just remove empty whitespace lines like this.

Comment (number5) [inline Library/Formula/luvit.rb]: will do :)

**Comment by DomT4** [comment]
How is this building correctly, out of interest? There's a whole thread of trying to get it to build in https://github.com/Homebrew/homebrew/pull/42525.

**Comment by mikemcquaid** [comment]
No idea.

**Pull request closed without merging**